src/resources/ApiKeys/ApiKeysInterfaces.ts

```diff
@@ -0,0 +1,78 @@
+export interface ApiKeyModel {
+    organizationId: string;
+    id: string;
+    enabled: boolean;
+    value: string;
+    displayName?: string;
+    description?: string;
+    createdBy?: UserModel;
+    createdDate?: string;
+    allowedIps?: string[];
+    apiKeysThatCanEdit?: IdAndDisplayNameModel[];
+    deniedIps?: string[];
+    groupsThatCanEdit?: IdAndDisplayNameModel[];
+    privileges: PrivilegeModel[];
+    resourceId?: string;
+}
+
+export interface UserModel {
+    additionalInformation?: {};
+    country?: string[];
+    credentialsExpired?: boolean;
+    displayName?: string;
+    email: string;
+    emailAliases?: string[];
+    emailConfirmed?: boolean;
+    enabled: boolean;
+    expired?: boolean;
+    firstName?: string;
+    lastLogin: number;
+    lastName?: string;
+    locked: boolean;
+    name?: string;
+    provider?: Provider;
+    providerUserId?: string;
+    providerUsername?: string;
+    realms?: UserRealmModel[];
+    samlIdentityProviderId?: string;
+    socialUser?: boolean;
+    username?: string;
+}
+
+export interface UserRealmModel {
+    member?: boolean;
+    realm?: RealmModel;
+}
+
+export interface RealmModel {

```

Comment:
only the id is required here

Reply:
in fact I don't think the user and realm should be defined in the api key api, it should be in the users api